### DISCUSSION TITLE
manual or simple or regular Cards as tabs

### DISCUSSION THREAD

**Discussion by Meet-Bhalodia** [ant-design/ant-design#44122] [General]
### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/antd-reproduction-template-forked-s3lw9m?file=/index.js)

### Steps to reproduce

visit codesandbox so you can understand what i actually expressing about.

### What is expected?

i need a a custom card as tab which can easy used for example onlicking tabs calling API as also in those card, The main reason is i need in grid which leads to responsive and good User experience, By these also i will be satisfied. JUST PLEASE VISIT CODESANDBOX 

### What is actually happening?

tabs are hidding and spoiling the UI i need a card as tabs which can easily used and manipulate as developers need

| Environment | Info |
| --- | --- |
| antd | 5.8.2 |
| React | react-js |
| System | windows 10 |
| Browser | chrome 115.0.5790.110 |

**Comment by MadCcc** [comment]
Did you mean that you don't want Tabs to be hidden here?
![image](https://github.com/ant-design/ant-design/assets/27722486/3c1c1d20-4e96-41a9-be4a-94f52261b93e)

**Comment by Meet-Bhalodia** [comment]
Ok, Now i am elobarating more about this, Exactly i need an customize cards grid means as i mentioned in codesandbox below Antd Tabs those are customized cards, actually i need multiple cards and those cards will contain different apis which will be requested on clicking as tabs works with different content.

Here is what is actually needs : [CodeSandBox](https://codesandbox.io/s/basic-usage-antd-5-8-2-forked-tk9zc6?file=/index.css)
i need these as **TABS** but its should be customized
_In This CodeSandBox there will be some css issue ignore and view in full screen **Open in New Window** to get better understanding of what i am actually needed and elaborating about_

I hope **Antd Team** will help me.

**Comment by Meet-Bhalodia** [comment]
Please help me @afc163 @zombieJ   , I am really stucked here.

**Comment by zombieJ** [comment]
Just customize a component which follow the API as Tabs if you want the similar API. Since this is not antd issue, let me move into Discussion instead.
